tools/cli/flags.go

```diff
@@ -226,6 +226,8 @@ const (
 	FlagFailoverTypeWithAlias             = FlagFailoverType + ", ft"
 	FlagFailoverTimeout                   = "failover_timeout_seconds"
 	FlagFailoverTimeoutWithAlias          = FlagFailoverTimeout + ", fts"
+	FlagRetryInterval                     = "retry_interval"
```

Comment:
maybe explicitly make it `retry_interval_in_sec` and add a alias?

Reply:
Other options don't seem to follow this convention, e.g. `execution_timeout` or `context_timeout` don't have explicit name. There are some that have `_seconds` suffix like `failover_timeout_seconds`. This makes me hesitant to add this third style `_in_sec`.   It is also somewhat more lengthy to type.